src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -1186,8 +1213,14 @@ public AutoEvoExploringToolWorld(IAutoEvoConfiguration configuration)
             foreach (var organelle in SimulationParameters.Instance.GetAllOrganelles())
             {
                 MicrobeSpeciesOrganelleStatistics.Add(organelle, (0, 0));
+                foreach (var upgrade in organelle.AvailableUpgrades.Keys)
+                {
+                    organelle.AvailableUpgrades.TryGetValue(upgrade, out var upgradeName);
+                    MicrobeSpeciesUpgradesStatistics.TryAdd(upgrade, (upgradeName?.Name ?? string.Empty, 0, 0));
```

Comment:
Shouldn't this have error handling and printing an error rather than adding an empty upgrade name? Doesn't it imply major json config error if the TryGetValue fails here?